ignite/cmd/plugin.go

```diff
@@ -19,29 +20,46 @@ import (
 	"github.com/ignite/cli/ignite/services/plugin"
 )
 
+const (
+	igniteCmdPrefix   = "ignite "
+	flagPluginsGlobal = "global"
+)
+
 // plugins hold the list of plugin declared in the config.
 // A global variable is used so the list is accessible to the plugin commands.
 var plugins []*plugin.Plugin
 
-const (
-	igniteCmdPrefix = "ignite "
-)
-
-// LoadPlugins tries to load all the plugins found in configuration.
-// If no configuration found, it returns w/o error.
+// LoadPlugins tries to load all the plugins found in configurations.
+// If no configurations found, it returns w/o error.
 func LoadPlugins(ctx context.Context, rootCmd *cobra.Command) error {
-	cfg, err := parseLocalPlugins(rootCmd)
-	if err != nil {
-		// if binary is run where there is no plugins.yml, don't load
+	pluginsConfigs := make([]pluginsconfig.Plugin, 0)
+
+	localCfg, err := parseLocalPlugins(rootCmd)
+	if err != nil && !errors.As(err, &cosmosanalysis.ErrPathNotChain{}) {
+		fmt.Printf("error parsing local chain plugins config: %s", err.Error())
 		return nil
```

Comment:
Since you filtered out the `cosmosanalysis.ErrPathNotChain{}`, you can probably return the error instead of nil.

Reply:
https://github.com/ignite/cli/pull/3214/commits/36db41933fcfde6ecf637138756c308f26562e65